### PR TITLE
Signer connection screen

### PR DESCRIPTION
**Pull request by ERussel**
- add mobile signer connection screen
- add connecting and request handling logic

### CODE REVIEW

File: fearless/Common/View/IconWithSubtitleView.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import UIKit
+
+class IconWithSubtitleView: UIView {
```
Comment (ef1rspb) [inline fearless/Common/View/IconWithSubtitleView.swift:3]: Is it a common view or closely related to Beacon feature?

File: fearless/Modules/Experimental/Signers/SignerConnect/SignerConnectViewController.swift [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+import SoraFoundation
+
+final class SignerConnectViewController: UIViewController, ViewHolder {
+    typealias RootViewType = SignerConnectViewLayout
+
+    let presenter: SignerConnectPresenterProtocol
+
+    private var iconViewModel: ImageViewModelProtocol?
+
+    init(presenter: SignerConnectPresenterProtocol) {
```
Comment (ef1rspb) [inline fearless/Modules/Experimental/Signers/SignerConnect/SignerConnectViewController.swift:11]: localizationManager?

File: fearless/Modules/Experimental/Signers/SignerConnect/SignerConnectViewLayout.swift [hunk 3]
```diff
@@ -0,0 +1,87 @@
+import UIKit
+
+final class SignerConnectViewLayout: UIView {
+    let contentView: ScrollableContainerView = {
```
Comment (ef1rspb) [inline fearless/Modules/Experimental/Signers/SignerConnect/SignerConnectViewLayout.swift:4]: simply `let contentView = ScrollableContainerView()` if no configuration applied

**Review by ef1rspb — APPROVED** [review]

File: fearless/Common/View/IconWithSubtitleView.swift (hunk 1 above)
Comment (ERussel) [reply fearless/Common/View/IconWithSubtitleView.swift:3]: I think it could be universal

**Pull request merged**